### ISSUE TITLE
Links broken in "From HTML to List of Words" (and others)

### ISSUE THREAD

**Issue by ianmilligan1** [programminghistorian/jekyll#277]
From one of our readers on the e-mail account: the Old Bailey Online print view that we use in a series of lessons, notably the HTML to List of Words ones (i.e. [1](http://programminghistorian.org/lessons/from-html-to-list-of-words-1) and [2](http://programminghistorian.org/lessons/from-html-to-list-of-words-2)) is broken.

The link at https://www.oldbaileyonline.org/print.jsp?div=t17800628-33 generates a 404 error.

**Comment by acrymble** [comment]
Thanks Ian. I'll have to go through those all soon and repair them. I know they're starting to creak.

**Comment by acrymble** [comment]
Ok this was caused by a crash of the Old Bailey site in August and a subsequent change to their URL structure which is also affecting #284 . I've used this as an excuse to go through and overhaul all of these python lessons so they work again. Fixed broken links, made compatible with python 3, fixed broken zip files, etc. Hopefully no new issues.

I'll have to add proper icons for the lessons page. But that will have to be another day.
